requirements.txt

```diff
@@ -12,6 +12,7 @@ stevedore>=1.20.0 # Apache-2.0
 aiohttp>=3.6.2
 aiohttp-apispec>=2.0.0
 
+python-magic-debian-bin>=0.4.25
```

Comment:
I think this should be just  `magic`, right?

https://pypi.org/project/python-magic/

Reply:
Using only magic in images such as bitnami / minideb, there are system libraries that are not installed as is the case of libmagic, with the python-magic-debian-bin package this problem is solved.

Reply:
Hmm, I understand. However, the problem is that this would only work with Debian derivates, not any other linux version. I guess we should leave the generic "python-magic" here.

Reply:
Ok, I change the library to python-magic in the requirements.txt file. Can I include the python-magic-debian-bin library in the test-requirements.txt file so that the tests don't fail?